azure_arc_k8s_jumpstart/docs/local_microk8s.md

```diff
@@ -0,0 +1,217 @@
+
+  az provider register --namespace Microsoft.KubernetesConfiguration
+  Registering is still on-going. You can monitor using 'az provider show -n Microsoft.KubernetesConfiguration'
+  ```
+  You can monitor the registration process with the following commands:
+  ```bash
+  az provider show -n Microsoft.Kubernetes -o table
+ 
+  az provider show -n Microsoft.KubernetesConfiguration -o table
+  ```
+
+* Install the Azure Arc for Kubernetes CLI extensions ***connectedk8s*** and ***k8sconfiguration***:
+
+  ```bash
+  az extension add --name connectedk8s
+  az extension add --name k8sconfiguration
+  ```
+
+# Deployment
+
+* Install MicroK8s following the [specific install guide](https://microk8s.io/) for your operating system. For convenience, we've added some commands below:
+
+  * Windows:
+  
+    Download the [MicroK8s installer for Windows](https://microk8s.io/docs/install-alternatives#heading--windows) and follow the instructions. 
+
+    ![](https://aws1.discourse-cdn.com/business6/uploads/kubernetes/original/2X/c/cc39a370d6a9f62bbcfa0b84ba8356da84a4c8c1.png)
+
+    Once installed, enable MicroK8s with:
+
+    ```terminal
+    microk8s status --wait-ready
+    microk8s enable dns
+    ```
+
+  * Linux:
+    ```terminal
+    sudo snap install microk8s --classic
+    microk8s status --wait-ready
+    microk8s enable dns
+    
+    ```
+  * MacOS:
+    ```
+    brew install ubuntu/microk8s/microk8s
+    microk8s install
+    microk8s status --wait-ready
+    microk8s enable dns
+    ```
+
+    ![](https://assets.ubuntu.com/v1/670398bd-mac1.png)
+
+  * WSL2
+
+    [This blog post](https://ubuntu.com/blog/kubernetes-on-windows-with-microk8s-and-wsl-2) walks through an installation of MicroK8s in WSL 2.
+
+* Navigate to the folder that has the MicroK8s cluster definition.
+
+  Windows:
+    ```terminal
+    cd azure_arc\azure_arc_k8s_jumpstart\microk8s
+    ```
+
+  Linux and MacOS:
+    ```bash
+    cd azure_arc/azure_arc_k8s_jumpstart/microk8s
+    ```
+
+* Export MicroK8s cluster kubeconfig file path
+
+  MicroK8s will not update your .kube/config file, and accessing the cluster is done using the microk8s cli, eg: `microk8s kubectl get nodes`. To be able to use this config with the Azure Arc CLI, we need to export it into a file.
+
+  Windows: 
+  ```
+  microk8s config view > %HOMEPATH%\.kube\microk8s
+  ```
+  Linux and MacOS: 
+
+  ```
+  microk8s config view > ~/.kube/microk8s
+  ```
+
+# Connect the cluster to Azure Arc
+
+* Authenticate Azure CLI
+
+  We'll start by authenticating our CLI with Azure, **replacing the values below** with the output from the command we issued to create service principal earlier (`az ad sp create-for-rbac`) in the prerequisite section of this guide.
+
+  ```
+  az login --service-principal -u appId -p password --tenant tenant
+  ```
+
+  ![](../img/local_microk8s/01.png)
+
+* Create a Resource Group
+
+  ```
+  az group create -n Arc-MicroK8s-Demo -l EastUS
+  ```
+
+  ![](../img/local_microk8s/02.png)
+
```

Comment:
Add a screenshot of an empty RG (mask subscription ID)

Reply:
@likamrat done